addons/mail/static/src/thread/message.xml

```diff
@@ -14,7 +14,7 @@
                 <span class="o-mail-own-name">
                     <strong class="o_redirect me-1 text-truncate cursor-pointer">
```

Comment:
should put conditional on `cursor-pointer` based on `disableActions` (also line 8 to consider)